docker-compose.dev.yaml

```diff
@@ -5,6 +5,7 @@ services:
   marketplace-server:
     image: kinecosystem/marketplace-server:latest
     command: ./tests/wait-for marketplace-internal:3000 -- npm run start
+    restart: no
```

Comment:
`no` translate to `false` in yaml, so you need parenthesis `"no"`. That's why travis is failing